src/components/DayPickerNavigation.jsx

```diff
@@ -219,6 +230,12 @@ export default withStyles(({ reactDates: { color, zIndex } }) => ({
     zIndex: zIndex + 2,
   },
 
+  DayPickerNavigation__bottom: {
```

Comment:
@ljharb I tried to avoid adding any styles that implicate layout to `DayPickerNavigation` for custom navigation, but this was the best I could think to come up with since any absolutely positioned children will be taken out of the flow—and then there won't be enough space under the calendar. any thoughts on this?